models/demos/mamba/README.md

```diff
@@ -22,13 +29,14 @@ To run the demo using custom input prompts, you can provide a different path to
 ```
 pytest --disable-warnings -q -s --input-method=json --input-path='path_to_input_prompts.json' models/demos/mamba/demo/demo.py
 ```
-Note: We currently only support json file with strictly 32 user prompts with same token length. Check the `models/demos/mamba/demo/prompts.json` file for reference.
 
----
+Any sequence length is supported. We currently only support JSON file with strictly 32 user prompts with same token length. Check the `models/demos/mamba/demo/prompts.json` file for reference.
+
+The prefill graph is not currently integrated into the demo. Therefore we currently process the prompt a single token at a time using the decode graph.
 
-Feel free to reach out if you have any questions or need further assistance!
+---
 
-## Mamba Unit Tests
+## Unit Tests
```

Comment:
These unit tests also runs prefill mode
We can tell users to select only decode mode by running tests as follow
```
pytest -k "ModelMode.DECODE" models/demos/mamba/tests/test_mamba_ssm.py
```

Reply:
I think its fine to have users run prefill tests